### PR TITLE
Use sys.executable instead of hard coding

### PR DESCRIPTION
**Pull request by nijkah**
## Description

Predefined 'python' can be different from the command which is really executed.
So replace 'python' with sys.executable

Fixes #(issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

### CODE REVIEW

**Review by msaroufim — APPROVED** [review]

**Comment by maaquib** [comment]
@nijkah Thanks for fixing this. Can you also verify the [sanity suite](https://github.com/pytorch/serve/blob/master/CONTRIBUTING.md) runs fine and then I can merge this

**Comment by nijkah** [comment]
@maaquib I thought `python torchserve_sanity.py` had been done in `AWS CodeBuild CI Report`s.
Should I run it in my environment? I tested it in the past, but it requires quite a lot of resources compared to my environment.

**Review by HamidShojanazeri — APPROVED** [review]

File: ts_scripts/install_dependencies.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import os
 import platform
 import argparse
+from shutil import which
```
Comment (mreso) [inline ts_scripts/install_dependencies.py:4]: This seems to be an unused import

**Review by mreso — APPROVED** [review]

**Review by msaroufim — CHANGES_REQUESTED** [review]
let
s address mathias feedback

**Pull request merged**